storage/block/types.go

```diff
@@ -185,6 +192,31 @@ type DatabaseBlock interface {
 
 	// Close closes the block.
 	Close()
+
+	// SetOwner sets the owner of the block
+	SetOwner(Owner)
+
+	// Owner returns the owner of the block
+	Owner() Owner
+}
+
+// databaseBlockPrivate is the private portion of the DatabaseBlock interface
+type databaseBlockPrivate interface {
```

Comment:
Does this interface need to exist btw?  Wired list could just concretely be tied to `*dbBlock`.  Unless this interface truly does help with testing, we could just tie it concretely to the type.

Reply:
It can't be tied to the concrete type because when you put things in the wired list you're dealing with an interface, not a concrete implementation.

Reply:
We could do a forced type assertion, but seems kind of gross